packages/grafana-runtime/src/services/SidecarService_EXPERIMENTAL.ts

```diff
@@ -104,6 +107,7 @@ export class SidecarService_EXPERIMENTAL {
     }
 
     this.memoryLocationService.push({ pathname: `/a/${pluginId}${path || ''}` });
+    reportInteraction('sidecar_service_open_app_v2', { pluginId });
```

Comment:
Could you explain why you changed from `sidecar_service_open_app` to `sidecar_service_open_app_v2`?

Reply:
> Could you explain why you changed from `sidecar_service_open_app` to `sidecar_service_open_app_v2`? If you want to highlight a difference between them, could you use the same event and add a property that differentiates them?

I can see one is deprecated and the other is not. Wdyt about it being :
```
reportInteraction('sidecar_service_open_app', {
 pluginId,
 version: 'current' | 'deprecated' }
 );
 ```

Reply:
Yeah not sure how to approach this. Like the deprecation is very pro format right now. Like there is not much using it and I will just change the code later on, just need to synchronize it as it means changing it in other apps just didn't want to do it now and spend more time on it.

But then I thought the naming of the event makes sense to be the same as the function itself and so here we are.

Like on some level maybe it makes sense to not differentiate them? Like if I later remove the old one and rename `sidecar_service_open_app_v2` => `sidecar_service_open_app`? Because don't think right now the distinction is even important just want to know if anybody is using it going forward.

Reply:
> Could you explain why you changed from sidecar_service_open_app to sidecar_service_open_app_v2?

I guess that is what you were asking about? Like why not just keep the event name the same in both versions?

Reply:
> > Could you explain why you changed from sidecar_service_open_app to sidecar_service_open_app_v2?
> 
> I guess that is what you were asking about? Like why not just keep the event name the same in both versions?

Yes! I was asking that.

> Yeah not sure how to approach this. Like the deprecation is very pro format right now. Like there is not much using it and I will just change the code later on, just need to synchronize it as it means changing it in other apps just didn't want to do it now and spend more time on it.
> 
> But then I thought the naming of the event makes sense to be the same as the function itself and so here we are.
> 
> Like on some level maybe it makes sense to not differentiate them? Like if I later remove the old one and rename sidecar_service_open_app_v2 => sidecar_service_open_app? Because don't think right now the distinction isn't even important, just want to know if anybody is using it going forward.

Regarding this, I'd go for using the same event:
1. If you want to compare the number of people using the deprecated code with the ones using the current, the data is in the same SQL, so it will be easier to get.
2. I'm not sure about the process to remove a tracking event. Not from the code, I mean from BQ and Rudderstack, so won't be better to unify it from the start and avoid removing it?

Reply:
Yes I agree will change it that way.

Reply:
I have a doubt: are you expecting to have that event in a code that will be deprecated in the short/middle term? In that case might be better to set 'v0/v1/v2...' as a value to the `version` prop rather than 'deprecated' and 'current'. 🤷  Up to you, you're the one with the most context 🙂

Reply:
> might be better to set 'v0/v1/v2...' as a value to the version

Yes I actually did that. There is a possibility there will be quite a few versions actually, ideally deprecated and removed quickly but having a version props in that case makes more sense.